journalpage/models.py

```diff
@@ -95,6 +95,7 @@ def journal_bibliographic_info_page(self, request, collection_acron3, acron):
             "acknowledgements": acknowledgements,
             "additional_information": additional_information,
             "digital_preservation": digital_preservation,
+            "digital_preservation_clockss": digital_preservation.filter(acronym="CLOCKSS"),
```

Comment:
@samuelveigarangel suspeito que isso nunca será None, porque ele retorna um queryset. Você teria que usar ou get no lugar de filter e tratar exceção ou adicionar um .first()

Reply:
Em um contexto bool, se um queryset for vazio, ele é interpretado com False.
> bool(). Testing a QuerySet in a boolean context, such as using bool(), or, and or an if statement, will cause the query to be executed. If there is at least one result, the QuerySet is True, otherwise False. For example:
```python
if Entry.objects.filter(headline="Test"):
    print("There is at least one Entry with the headline Test")
```